src/index.ts

```diff
@@ -172,9 +172,18 @@ export const main = async (unresolvedConfiguration: CommandLineOptions) => {
           const isHandled = deputy.maybeAddReferencedExternalDependency(workspace, packageName);
           if (!isHandled) collector.addIssue({ type: 'unlisted', filePath: containingFilePath, symbol: specifier });
 
-          // Pattern: @local/package/file
+          // Pattern: self-reference/import
```

Comment:
Just to be sure I understand: the changes in this file are to accommodate self-references from dependencies coming from plugins, right? I mean this does not seem to affect the fixture added in this PR.

Reply:
I've added a new test fixture for this.